docs/src/user-guide/core-concepts.md

```diff
@@ -12,75 +12,72 @@ This page contains a high-level overview of some of the core concepts of ESLint.
 
-ESLint contains hundreds of built-in rules that you can use.
-You can also create custom rules or use rules that others have
-created with [plugins](#plugins).
+ESLint contains hundreds of built-in rules that you can use. You can also create custom rules or use rules that others have created with [plugins](#plugins).
 
 For more information, refer to [Rules](../rules/).
 
 ## Configuration Files
 
-An ESLint configuration file is a single place where you put all the configuration
-for ESLint your project. You can include built-in rules, how you want them enforced,
-plugins with custom rules, which files you want rules to apply to, and more.
+An ESLint configuration file is a place where you put the configuration for ESLint in your project. You can include built-in rules, how you want them enforced, plugins with custom rules, shareable configurations, which files you want rules to apply to, and more.
 
-For more information, refer to [Configuration Files](./configuring/configuration-files.md).
+For more information, refer to [Configuration Files](./configuring/configuration-files).
+
+## Shareable Configurations
+
+Shareable configurations are ESLint configurations that are shared via npm.
+
+Often shareable configurations are used to enforce style guides using ESLint's built-in rules. For example the sharable configuration [eslint-config-airbnb-base](https://www.npmjs.com/package/eslint-config-airbnb-base) implements the popular Airbnb JavaScript style guide.
+
+For more information, refer to [Using a shareable configuration package](./configuring/configuration-files#using-a-shareable-configuration-package).
 
 ## Plugins
 
-An ESLint plugin is an npm module that contains a set of ESLint rules.
-Plugins can be used to enforce a style guide and support JavaScript extensions
-(like TypeScript) or frameworks (like React).
-You can add single rules or sets of rules from a plugin.
-Often plugins come with recommended sets of rules that you can include in your project.
+An ESLint plugin is an npm module that can contain a set of ESLint rules, configurations, processors, and environments. Often plugins include custom rules. Plugins can be used to enforce a style guide and support JavaScript extensions (like TypeScript), libraries (like React), and frameworks (Angular).
+
+A popular use case for plugins is to enforce best practices for a framework. For example, [@angular-eslint/eslint-plugin](https://www.npmjs.com/package/@angular-eslint/eslint-plugin) contains best practices for using the Angular framework.
 
 For more information, refer to [Configuring Plugins](./configuring/plugins.md).
 
+## Parsers
+
+An ESLint parser converts code into an abstract syntax tree that ESLint can evaluate. By default, ESLint uses a built-in parser that is compatible with standard JavaScript runtimes and versions.
```

Comment:
Might be worth mentioning the default parser is Espree and link to the eslint/espree repo in this paragraph.